draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -1148,6 +1165,8 @@ Relying parties with additional sources of revocation such as {{CRLite}} or {{CR
 
 The transparency service does not prevent unauthorized certificates, but it aims to provide comparable security properties to Certificate Transparency {{?RFC6962}}. If a subscriber presents an acceptable Merkle Tree certificate to a relying party, the relying party should have assurance it was published in some form that monitors and, in particular, the subject of the certificate will be able to notice.
 
+One notable difference with Certificate Transparency is that transparency services do not publish the public keys, but rather hashes of them. This is intended to reduce serving costs, particularly with large post-quantum keys. Relying partings can still detect misissuance by looking for unexpected `subject_info_hash` values. However, this optimization complicates studies of weak public keys, e.g. {{SharedFactors}}. Such studies will have to retrieve the public keys separately, such as by connecting to the TLS servers, or fetching from the CA if it retains the unabridged assertion. This document does not define a mechanism for doing this.
```

Comment:
Oops, I typed "Relying partings" in my suggestion. Let's see if I can just fix that inline...